azure-container-apps/ReadMe.md

```diff
@@ -51,33 +51,29 @@ Deploying 1Password SCIM Bridge on Azure Container Apps comes with a few benefit
 
 ## Deploy 1Password SCIM Bridge to Azure Container App
 
-### Azure Container App Deployment steps using Azure CLI
+### Azure Container App Deployment steps using the Azure Shell
 
 Using the Azure Cloud Shell can be easier (the assumption for the commands below is that you are using the Cloud Shell), but this can also been done directly on your system with the Azure CLI tool with the ContainerApp Extension
 
 Both methods need the Container App Extension added to the Azure tool of choice, as detailed in step two.
 
 1. Start the Azure Cloud Shell from the navigation bar of your [Azure Portal](https://portal.azure.com) or directly open the [Azure Shell](https://shell.azure.com).
 2. Add the Azure Container App extension by running the following command: `az extension add --name containerapp --upgrade`
-3. Get a list of the available locations in your Azure account: `az account list-locations -o table`, verify that the region you want to deploy on [supports Azure Container Apps](https://azure.microsoft.com/en-ca/explore/global-infrastructure/products-by-region/?regions=all&products=container-apps), take note of the name field for the desired region. 
+3. Get a list of the available locations in your Azure account: `az account list-locations -o table`, take note of the name field for the desired region. 
 
     > **Note:** This command can be skipped if you already know the region you want to deploy to and if that region supports Container Apps. 
```

Comment:
Do all regions now support container apps? If so, this note should be updated too.

Reply:
Not all regions support Container Apps still. The amount of regions has increased drastically. The wording that will end up in AKB is slightly different, and once that ends up in AKB, this section with the deployment steps will be removed from this readme.

"Find your preferred server location (region) for the deployment and make note of its `Name`, such as `eastus`." is the wording that will be used and no mention in AKB about the supported regions. This is why we have added a section to readme in [Troubleshooting for Region Support](https://github.com/1Password/scim-examples/tree/promote/azure-container-apps/azure-container-apps#region-support).